lib/component.js

```diff
@@ -161,6 +161,21 @@ class Component extends WebComponent {
     }
   }
 
+  /**
+   * Returns the reference to the parent of the given name. An error is thrown out if no parent of
+   * such name is found.
+   * @param {string} tagName - name of the parent to search for.
+   * @example this.parentbyTagName('my-app');
```

Comment:
no, i was just pointing out that the capitalization in the doc didn't match the actual method name ('parentbyTagName' vs 'parentByTagName'), sorry about the confusion